charts/lightdash/values.yaml

```diff
@@ -213,6 +213,7 @@ serviceAccount:
 ## Customise backend configuration for GCP
 backendConfig:
   create: false
+  terminationGracePeriodSeconds: 90
```

Comment:
this is misleading it's for the GCP backendConfig (i.e. for loadbalancing) not for the api backend